packages/jaeger-ui/src/components/TracePage/index.tsx

```diff
@@ -280,6 +280,9 @@ export class TracePageImpl extends React.PureComponent<TProps, TState> {
     if (this.props.trace && this.props.trace.data) {
       this.traceDagEV = calculateTraceDagEV(this.props.trace.data);
     }
+    if (traceGraphView) {
+      this.focusUiFindMatches();
+    }
```

Comment:
This seems kind of strange. So, it will focus the UI find matches when you go back to the timeline view?

Reply:
I thought I had removed this... It doesn't work without more changes and as you said it may not be the desired functionality.
Will really remove and push.